templates/components/cv_overview_table.md

```diff
@@ -1,5 +1,5 @@
 |       | Substrate | Electrolyte {: .echemdb-expand-column } | Year {: .echemdb-hide-mobile } | Reference {: .echemdb-hide-mobile } |
 | ----- | --------- | ----------- | ---- | --------- |
 {% for entry in database %}
-| [![{{ entry.identifier}}](data:image/png;base64,{{ entry.thumbnail(96, 72) | b64encode }}){: .echemdb-cv-thumbnail }](entries/{{ entry.identifier }}) {: .echemdb-middle-cell } | {{ material }}({{ entry.system.electrodes.working_electrode.crystallographic_orientation }}) {: .echemdb-middle-cell } | {{ entry.system.electrolyte | render("components/electrolyte.md") }} {: .echemdb-middle-cell } | {{ database.bibliography.entries[entry.source.citation_key].fields['year'] }} {: .echemdb-hide-mobile .echemdb-middle-cell } | [ {{ entry.bibliography.persons['author'][0].last_names[0] | unicode }} ***et. al.*** Fig. {{ entry.source.figure }} ({{ entry.source.curve }})]({{ entry.source.url }}) {: .echemdb-hide-mobile .echemdb-middle-cell } |
+| [![{{ entry.identifier}}](data:image/png;base64,{{ entry.thumbnail(96, 72) | b64encode }}){: .echemdb-cv-thumbnail }]({{ entries_path }}/{{ entry.identifier }}) {: .echemdb-middle-cell } | {{ material }}({{ entry.system.electrodes.working_electrode.crystallographic_orientation }}) {: .echemdb-middle-cell } | {{ entry.system.electrolyte | render("components/electrolyte.md") }} {: .echemdb-middle-cell } | {{ database.bibliography.entries[entry.source.citation_key].fields['year'] }} {: .echemdb-hide-mobile .echemdb-middle-cell } | [ {{ entry.bibliography.persons['author'][0].last_names[0] | unicode }} ***et. al.*** Fig. {{ entry.source.figure }} ({{ entry.source.curve }})]({{ entry.source.url }}) {: .echemdb-hide-mobile .echemdb-middle-cell } |
```

Comment:
I decided to choose this option instead of 

`use_directory_urls: False` in the mkdocs yaml.

With that option turned on, the pages need to end in .html, which has to be considered when links are created. on the other hand this will only solve the problem temporarily, since such tabels can easily be generated in any kind of subfolder.

Reply:
Sure. We can just use `/cv/entries/...` though it seems which renders as an absolute link.

I pushed some (overengineered) attempt to render this correctly with relative links but I don't think it's worth it. We (and others) probably never want to deploy this to https://my.domain.example/base_path/ with a non-empty base path.